app/guides/layout.tsx

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+import { getGuidePages } from "@/lib/server/cms-data";
+import { getSections } from "@/lib/nav-menu-builder";
+import NavMenu from "@/components/NavMenu";
+
+export default async function GuidesLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const guidePages = await getGuidePages();
```

Comment:
the `NavMenu` can make this call instead of the layout. that way the data needed is local to the file it is being used in

Reply:
So I don't know how I feel about this.. Do we want the `NavMenu` to contain page specific code? I figured it'd be best to make the call outside the nave menu and then pass the options in only. Makes it more generic and we might use it elsewhere, no?

Reply:
I think what KRM is suggesting, based on the file name this is a display component as it's `Layout` it should stay dumb and we should move the handling of logic/data fetching to a smart component. As `guidePages` is only consumed by the `NavMenu` - moving the logic there for now makes sense